7_line_following/main/line_following.c

```diff
@@ -5,15 +5,16 @@
 #include "tuning_http_server.h"
 
 #define MODE NORMAL_MODE
-#define BLACK_MARGIN 400
-#define WHITE_MARGIN 2000
+#define BLACK_MARGIN 4095
+#define WHITE_MARGIN 0
 #define bound_LSA_LOW 0
 #define bound_LSA_HIGH 1000
+#define BLACK_BOUNDARY 700
 
 /*
  * weights given to respective line sensor
  */
-const int weights[4] = {3,1,-1,-3};
+const int weights[5] = {3, 1, 0, -1, -3};
```

Comment:
if the middle sensor's weight is not taken into account then what is the purpose of adding an additional sensor?

Reply:
Got it.I will make the required changes with new weights and issue a new PR.